src/go/rpk/pkg/cli/cmd/redpanda/tune/tune.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Redpanda Data, Inc.
+// Copyright 2022 Redpanda Data, Inc.
```

Comment:
I think technically we don't need to update these. OK for this PR, but just fyi for future PRs.